### DISCUSSION TITLE
8月14号的版本，导线的链接会在区块重新加载时失效，重新放导线就会恢复

### DISCUSSION THREAD

**Discussion by goingupbao** [IndustrialCraftMC/IndustrialCraft2#103] [【漏洞反馈】Bugs Report]
偶尔启动崩溃，不知道跟导线的bug是不是有关
另外还没有 “工具箱” 金属成型机无法合成，
采矿机无法自动输出到旁边的箱子（忘记老版本是不是这样的了）

风力动能发电机 传送机 提取机 电炉 方块切割机 热能离心机   金属成型机 在jei下没有配方

**Comment by goingupbao** [comment]
[minecraft-exported-crash-info-2023-08-24T21-15-16.zip](https://github.com/IndustrialCraftMC/IndustrialCraft2/files/12440718/minecraft-exported-crash-info-2023-08-24T21-15-16.zip)\

搞了一个弹出升级给采矿机，崩溃了，现在进游戏就崩溃

**Reply by AbbottMc** [reply]
已修复

**Reply by goingupbao** [reply]
下载了新版，弹出升级的好了，这个导线的问题还是依旧哦，区块重新加载，导线的连接就会失效

**Reply by Stephen633** [reply]
我的版本是ic2-forge-unspecified+551d017
铜质导线（有绝缘）依然有导线的bug，需要有重新摆放， 电锯，工具箱没法合成，流体和普通弹出升级在我的客户端可以合成，但是没有功能

mod 表
ic2
豆腐craft
暮色森林
FarmersDelight
还有一些辅助性的mod（小地图，HUD，JEI一类的）

**Reply by goingupbao** [reply]
我换了fabric的版本没有导线的问题了，看来fabric的进度比较超前
